app/models/spree/subscription_frequency.rb

```diff
@@ -8,8 +8,15 @@ class SubscriptionFrequency < Spree::Base
     validates :title, :months_count, presence: true
     with_options allow_blank: true do
       validates :months_count, numericality: { greater_than: 0, only_integer: true }
-      validates :title, uniqueness: { case_sensitive: false }
     end
+    validate :validate_max_months_count
 
+
+    def validate_max_months_count
```

Comment:
i think we don't need to write a seperate method. you can handle this within existing
``` validates :months_count, numericality: { greater_than: 0, only_integer: true }```